src/migrations/index.js

```diff
@@ -1,4 +1,4 @@
-import { r, Migrations } from '../server/models'
+import { r, Migrations, CannedResponse } from '../server/models'
```

Comment:
Let's remove CannedResponse here -- I don't think it's required and it's possible it will subtly affect import order.